src/util.ts

```diff
@@ -147,7 +147,7 @@ export function infoResponse(
  */
 export function parseGitName(repo: string): string {
   const locationOfGit = repo.lastIndexOf('.git');
-  return repo.substr(0, locationOfGit > 0 ? locationOfGit : repo.length);
```

Comment:
why do we want to use slice here?

Reply:
`String.substr` is deprecated, so I replaced it with `slice` instead as with 0 being the first argument they're doing the same thing